Automattic/wp-calypso
# Re-enable private by default in development, test, & wpcalypso environments

IMPORTANT: When this lands, update P9HQHe-k8-p2 !

This is the "main branch" to enable the Private-by-Default (PBD) feature.

Lots of things have been merged in here:
* Revert of the revert
* Wrap in a feature flag (enabled in development & wpcalypso environments)
* Add an AB test (90% selected / 10% control)
* Sites created under a Business, eCommerce(, & any other other auto-Atomic) plans will be created as public (for now -- paObgF-pN-p2 )
* Launch site / set public when upgrading to an auto-atomic plan

See p7DVsv-6YR-p2

### To Test

* Run this branch (calypso.live should work as well)
* Browse to `/start`
* Mouse over the "environment badge"
* Mouse over `ABTESTS`
* Scroll down the list of active tests until you see `privateByDefault`
* Click the radio button for `selected`
* Create a site under any segment except for business or online-store
  * It should be unlaunched ("private by default")
* Follow the test plan in #35003 to test Upgrades
* Disable the config flag `private-by-default/non-atomic-plans` & repeat -- all sites should be created as public & no new error conditions should be achievable.

Fixes #32671
